intake_stac/drivers.py

```diff
@@ -0,0 +1,217 @@
+from intake.source.base import DataSource, PatternMixin, Schema
+from intake.source.utils import reverse_formats
+from pkg_resources import get_distribution
+
+__version__ = get_distribution('intake_stac').version
+
+
+class RioxarraySource(DataSource, PatternMixin):
```

Comment:
intended to be a simplified version of https://github.com/intake/intake-xarray/blob/master/intake_xarray/raster.py seeing what is the minimal driver if intake-xarray were to be separated out per xarray backends (netcdf, zarr, raster, etc)